test/assets/date-picker-utils.test.js

```diff
@@ -1,4 +1,7 @@
-const { expect } = require('test/util/chai');
+// const { expect } = require('test/util/chai');
```

Comment:
Didn't seem to agree with this proxy import. Shall double check then probably remove.